src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar search-bar--big"  data-qa="big">
+      <input
+        class="search-bar__input"
+        type="search"
+        placeholder="Try “Los Angeles”"
+      >
+    </div>
+    <div class="search-bar search-bar--small"  data-qa="small">
+      <input
+        class="search-bar__input"
+        type="search"
+        placeholder="Try “Los Angeles”"
+        >
+    </div>
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91411692/92c43806-c7ee-45cf-9cf8-859035762621)

```suggestion
      <input
        class="search-bar__input"
        type="search"
        placeholder="Try “Los Angeles”"
      >
    </div>
```